internal/kafka/consumer.go

```diff
@@ -59,6 +57,9 @@ func StartConsumer(kafka_topic string, handler func(msg *kafka.Message, consumer
 		}
 	}
 
+	configMap["session.timeout.ms"] = 120000
+	configMap["heartbeat.interval.ms"] = 30000
```

Comment:
Could we move these inside the configMap var on L#51?

Reply:
No, we can't because when PROD will move to AWS Managed Kafka(migration on 27th June) then these config will not get applied if we move these changes to L#51.

Reply:
Ah the else clause, my bad